framework/framework.go

```diff
@@ -0,0 +1,8 @@
+package framework
+
+// Flag is used by many of the framework generators
+type Flag struct {
```

Comment:
This might not be the best place for this, but I basically needed configuration I could share to a lot of packages that doesn't have a lot of imports avoiding circular dependencies.